pkg/ytconfig/node.go

```diff
@@ -325,9 +327,11 @@ func getDataNodeServerCarcass(spec *ytv1.DataNodesSpec) (DataNodeServer, error)
 
 			// These are just simple heuristics.
 			gb := float64(1024 * 1024 * 1024)
-			storeLocation.LowWatermark = int64(math.Min(0.1*float64(storeLocation.Quota), float64(5)*gb))
+			storeLocation.LowWatermark = int64(math.Max(float64(location.Watermark)/100*float64(storeLocation.Quota), float64(5)*gb))
 			storeLocation.HighWatermark = storeLocation.LowWatermark / 2
 			storeLocation.DisableWritesWatermark = storeLocation.HighWatermark / 2
+			storeLocation.TrashCleanupWatermark = storeLocation.LowWatermark
```

Comment:
Does it make sense to have those two fields with the same value? Is it better if we explicitly put it in existing cluster dnds configs?